### ISSUE TITLE
PhantomJS itself should not be a requirement during module installation

### ISSUE THREAD

**Issue by seelts** [amir20/phantomjs-node#581]
While installing this module, it tries to download and install the PhantomJS, because it is not in my PATH.

This seems to be a wrong behaviour.
I already have PhantomJS, but I don't want it to be in PATH. I'm going to point the full path to it inside node.js script.

Also, in my case the installer wasn't even able to finish the process, because bzip2 is not available.

PhantomJS should be just a runtime requirement.

Linux inside docker.

**Comment by amir20** [comment]
@seelts  This question has been asked many times. For a while, I was convinced that it should not be part of this module to install phantomjs. But after a bunch of builds breaking because phantomjs was not coupled, I decided to include it. It is also a [12factor principal](https://12factor.net/dependencies) to include all dependencies. 

At the end, I evaluated which is the most common use case and how I can get people to start using this package really easy. Including the prebuilt package was the right choice for that. 

I have been able to get this module to build with the following Docker file. 

```
FROM ubuntu:15.10
MAINTAINER Amir Raminfar <findamir@gmail.com>

RUN apt-get update && apt-get install -y curl
RUN curl -sL https://deb.nodesource.com/setup_5.x | bash -
RUN apt-get install -y nodejs build-essential g++ flex bison gperf ruby perl \
  libsqlite3-dev libfontconfig1-dev libicu-dev libfreetype6 libssl-dev \
  libpng-dev libjpeg-dev python libx11-dev libxext-dev

RUN npm install phantom co

COPY simple.js simple.js
```

I'll leave this ticket open for one day to get your response, otherwise no actionable item is required.
